lib/ansible/modules/windows/win_xml.py

```diff
@@ -85,9 +92,4 @@
     returned: always, for type element and -vvv or more
     type: list
     sample: attribute mismatch for actual=string
-backup:
```

Comment:
I would keep this here for now and say use backup_file and this is deprecated.

Reply:
Fixed.